content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -363,6 +366,44 @@ will be terminated once the job backoff limit has been reached. This can make de
 from failed Jobs is not lost inadvertently.
 {{< /note >}}
 
+### Backoff limit per index {#backoff-limit-per-index}
+
+{{< feature-state for_k8s_version="v1.28" state="alpha" >}}
+
+{{< note >}}
+You can only configure the backoff limit per index for a Job, if you
+have the `JobBackoffLimitPerIndex` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+enabled in your cluster. A per-index backoff limit is also only available for indexed Jobs that
+use the `restartPolicy: Never`.
+{{< /note >}}
+
+When you run an [indexed](#completion-mode) Job, you may want to handle pod failures for different
+indexes as counted independently. To do so, set the `.spec.backoffLimitPerIndex`
+to specify the the maximal number of pod failures per index. For example, to allow exactly
+one retry for each indexed subtask in a Job and to fail early if the retry also fails, set `.spec.backoffLimitPerIndex` to 1 and set `.spec.maxFailedIndexes` to 0.
+
+When the per-index backoff limit is
+exceeded for an index, the index is marked as failed in the
+`.status.failedIndexes` field. Note that, a failing index does not interrupt
+execution of other indexes. You can additionally limit the maximal number of
+indexes marked failed by setting the `.spec.maxFailedIndexes` field. In that case,
+the the number of failed indexes is exceeded, then the entire Job is marked
+failed; marking that failure triggers termination of all running Pods for that Job.
+
+If all indexes finish for a Job where you specified a per-index backoff limit, and at
+least one of those indexes did fail, but there wasn't an early termination (due to
+reaching `.spec.maxFailedIndexes`), then at this point the Kubernetes control plane
```

Comment:
it doesn't matter if it reaches the maxFailedIndexes or not. All that matters is that at least one Index failed.

Reply:
Yes, remove that part.